src/main/java/sirius/biz/jobs/BasicJobFactory.java

```diff
@@ -210,6 +225,11 @@ public void startInteractively(WebContext request) {
         });
 
         if (submit.get()) {
+            if (TenantController.isCurrentlySpying(request) && shouldExecuteInAdminTenant()) {
```

Comment:
interative jobs don't matter as they don't create processes for other tenants...